pallets/xyk/src/lib.rs

```diff
@@ -431,15 +451,12 @@ decl_module! {
     }
 }
 
-
 impl<T: Trait> Module<T> {
```

Comment:
set formatter to ignore empty lines, we should have them to visually split the code